.gitignore

```diff
@@ -19,5 +19,8 @@ venv.bak/
 *.idea
 *.iml
 
-# Outputs
-outputs/
\ No newline at end of file
+# Data
+data/
```

Comment:
change data/ to /data/, as we don't want to ignore data directory in folders such as tests (https://www.atlassian.com/fr/git/tutorials/saving-changes/gitignore)